lab01-activity.Rmd

```diff
@@ -150,9 +153,13 @@ The second line should be:
 
 Be careful here with the distinction between `>` and `>>`. The code should only "append" lines to the file if/when necessary (i.e. when re-run you will not get additional lines in the file). You can check that your code is working by running the chunk more than once and then verifying the contents of the file. If you are so inclined, you can even add a line to your code chunk that will report the current number of lines in the file each time the code chunk is run. Give it a try if you're feeling ambitious! By the end of this lab you will need to have completed this such that you have a bash_profile file with those two lines in it. See the instructor or TA for help if you don't think you succeeded. The easist way to confirm is to run this command in your terminal (not in Rmarkdown) and looking at the contents:
 
-`cat ~/.bash_profile`
+`cat ~/.bash_profilecat ~/.bash_profile`
+
+```{bash}
+echo "source ~/.bashrc" >> ~/.bash_profile
```

Comment:
Completed
Luc, > is used for overwriting and when you re-run the code, you'll have one source ~/.bashrc
>> is used for appending and when you re-run it each time it adds a new line to the previous ones.